front/app/containers/LandingPage/EventsWidget/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+// components
+import TopBar from './TopBar';
+import EventCard from 'components/EventCard';
+import EventsError from 'containers/EventsPage/EventsViewer/EventsError';
+import EventsSpinner from 'containers/EventsPage/EventsViewer/EventsSpinner';
+import VerticalCenterer from 'components/VerticalCenterer';
+
+// hooks
+import useEvents from 'hooks/useEvents';
+
+// i18n
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+
+// styling
+import styled from 'styled-components';
+import { colors, fontSizes } from 'utils/styleUtils';
+
+// other
+import { isNilOrError, isNil, isError } from 'utils/helperUtils';
+import messages from 'containers/EventsPage/messages';
+
+const EventsWidgetContainer = styled.div`
+  padding: 48px 0 124px 0;
+`;
+
+const NoEventsText = styled.div`
+  margin: auto 0px;
+  text-align: center;
+  color: ${colors.label};
+  font-size: ${fontSizes.xl}px;
+`;
+
+const CardsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  > * {
+    margin: 0px 19px;
+  }
+  > :first-child {
+    margin-left: 0px;
+  }
+  > :last-child {
+    margin-right: 0px;
+  }
+`;
+
+const StyledEventCard = styled(EventCard)`
+  border-radius: 3px;
+  padding: 20px;
+
+  > div > div > h3 > span {
```

Comment:
This seems really weird, it's very much linked to the internals of the component it's styling, I think you should achieve this in another way. It can be adding a class on the span in the event card and using it here to style it, or defining a prop to control the different font sizes that an event can show the content of their > div > div > h3 > span elements.